app/models/event.rb

```diff
@@ -224,19 +233,26 @@ def calculate_progress
   end
 
   ##
-  # Returns end of the event
+  # Returns the room in which the event is scheduled
+  #
+  def scheduled_room
```

Comment:
Because there are many places where we did `event.room` so doing `event.scheduled_room` we keep everything the same. And at the beginning the function `event_schedule` was private as it was only used for the `scheduled_room` and `scheduled_start_time `but now I am using it in more places and it is public. So, I can eliminate `scheduled_room` and `scheduled_start_ time` functions and do everywhere `event. event_schedule.try(:room)`. Do I change it then?